.eslintrc.js

```diff
@@ -0,0 +1,30 @@
+module.exports = {
```

Comment:
This is fairly minimal at the moment, copied from https://github.com/mitodl/ol-npm-libraries/blob/main/.eslintrc.js . 
 (I would like to update `eslint-config-mitodl` and move it into that repo.)

This does NOT affect linting in `frontends/open-discussions`, since that has its own eslint which specifies `root: true`